_translator-files/po/es/1-index.po

```diff
@@ -128,19 +107,19 @@ msgstr "¿Quieres involucrarte?"
 #: ../wiki/en/misc/1-index.md:53
 #, no-wrap
 msgid "Ideas? Found a bug? Want to contribute some code or help [translating](https://github.com/jamulussoftware/jamulus/blob/main/docs/TRANSLATING.md) Jamulus into your language? Since Jamulus is [free and open source software](https://www.gnu.org/philosophy/free-sw.en.html) (FOSS) licensed under the [GPL](https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html) you can help us!"
-msgstr ""
+msgstr "¿Ideas? Has encontrado un bug? ¿Quieres contribuir con código o ayudar a [traducir](https://github.com/jamulussoftware/jamulus/blob/main/docs/translating.md) Jamulus a tu idioma? Puesto que Jamulus es [software libre y de código abierto](https://www.gnu.org/philosophy/free-sw.en.html) (FOSS) bajo la licencia [GPL](https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html) ¡nos puedes ayudar!"
 
 #. type: Content of: <div><div><p><div><br><br>
 #: ../wiki/en/misc/1-index.md:55
 #, no-wrap
 msgid "Take a look at our [contribution guidelines](wiki/Contribution) to find out how. Everybody is welcome!"
-msgstr ""
+msgstr "Echa un vistazo a nuestra [guía para contribuciones](wiki/Contribución) para saber cómo. ¡Cualquier persona es bienvenida!"
```

Comment:
Not sure if that's the correct link:

```suggestion
msgstr "Echa un vistazo a nuestra [guía para contribuciones](wiki/Contribution) para saber cómo. ¡Cualquier persona es bienvenida!"
```

Reply:
Thanks - corrected. There's a user on Weblate who has the exasperating habit of changing stuff that was already ok. He's well-intentioned but it's exhausting, as I have to go after him and comb through everything he touches... I've repeatedly told him to leave links alone, to not arbitrarily change things on a whim without any consistency... If it keeps going on I'll have to end up blocking him.